tests/rst_output_pytest.py

```diff
@@ -385,3 +385,109 @@
     "| :gherkin-step-keyword:`Then` the file is converted into rST\n",
     "\n",
 ]
+
+no_background_rst = [
+    ".. role:: gherkin-step-keyword\n",
+    ".. role:: gherkin-step-content\n",
+    ".. role:: gherkin-feature-description\n",
+    ".. role:: gherkin-scenario-description\n",
+    ".. role:: gherkin-feature-keyword\n",
+    ".. role:: gherkin-feature-content\n",
+    ".. role:: gherkin-background-keyword\n",
+    ".. role:: gherkin-background-content\n",
+    ".. role:: gherkin-scenario-keyword\n",
+    ".. role:: gherkin-scenario-content\n",
+    ".. role:: gherkin-scenario-outline-keyword\n",
+    ".. role:: gherkin-scenario-outline-content\n",
+    ".. role:: gherkin-examples-keyword\n",
+    ".. role:: gherkin-examples-content\n",
+    ".. role:: gherkin-tag-keyword\n",
+    ".. role:: gherkin-tag-content\n",
+    "\n",
+    ":gherkin-feature-keyword:`Feature:` :gherkin-feature-content:`Testing Sphinx Writer`\n",
+    "====================================================================================\n\n",
+    "    :gherkin-feature-description:`Test the ability to write Gherkin data to a SphinxWriter object,`\n",
+    "    :gherkin-feature-description:`with a multiline feature description for pytest.`\n",
+    "\n",
+    "\n",
```

Comment:
I'm a bit confused, wasn't there just one blank line in the source feature file between these paragraphs?

Reply:
There was -- our current logic still sees the `"."` trailing a line as a trigger to add another return. Ryan and I discussed and this is a "good enough" solution for now, with room to come back with another PR soon-ish to better separate the description parsing to custom-fit for the `behave` vs `pytest-bdd` cases.